scripts/plot_radiosonde_data.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Plot radiosonde level 1 data
+
+Created on Sun Jan 29, 2023
+
+@author: laura
+
+run the script:
+    python make_plots_radiosonde.py -i inputfilename -o outputdirectory
+for level 1 radiosonde files. The inputfilename should include the direction (ascent or descent). If not, no SkewT diagram is made.
+
+This script produces 3 plots:
+    - trajectory
+    - temperature T, dew point tau, relative humidity rh, water vapor mixing ratio, wind speed, and wind direction
+    - SkewT diagram (only if direction, i.e. ascent or descent, is given in inputfilename)
+"""
+
+import argparse
+import os.path
+
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import metpy.calc as mpcalc
+import numpy as np
```

Comment:
```suggestion
import numpy as np
import pint_xarray
```
Package to create unit aware datasets